.github/workflows/build-conformant-images.yml

```diff
@@ -89,7 +89,7 @@ jobs:
 
     create-image-all-services-s390x:
         strategy:
-            max-parallel: 2
+            max-parallel: 1
```

Comment:
it looks like this didn't help, shall we revert it?

Reply:
@achmelo this never actually took affect, the `build-conformant-images.yml` on `master` is used to run the workflow, the `max-parallel: 1` change was done on `v2.x.x`.

I can guarantee this will help at least a tiny bit, the question is if the help is at all significant and if it's worth doubling the time to build s390x